src/main/scripts/cbas

```diff
@@ -29,12 +29,10 @@ def get_jump_url(config):
     Return the jumphost url with HTTP scheme if we're in testing mode,
     HTTPS otherwise.
     """
-    if "://" in config.jump_host:
-        return config.jump_host
-
-    if config['password_provider'] == 'testing':
-        return 'http://' + config.jump_host
-    return 'https://' + config.jump_host
+    return (config.jump_host
+            if "://" in config.jump_host
+            else 'https://{0}'.format(config.jump_host)
+            )
 
```

Comment:
I think that the explicit if is easier to read than the inline if, at least in this specific case.